AndelaOSP/wire
# #164721772 Fix material ui components on the admin dashboard

#### What does this PR do?
Fixes the outdated button and select material-ui components in the settings page

####
- replace the invite button with a new button
- replace the country filter button
- replace the position changer select field


#### How should this be manually tested?
- Navigate to the settings page by clicking on the profile image and selecting settings
-  click on the invite button, the location selector and the role selector buttons,
- They should not display the black colour previously displayed but should remain as they are

#### What are the relevant pivotal tracker stories?
[164721772](https://www.pivotaltracker.com/story/show/164721772)
#### Screenshots (if appropriate)
N/A
